php-practice_1.php

```diff
@@ -1,41 +1,154 @@
-<?php
 // Q1 変数と文字列
+<?php
+
+$name = '安藤';
 
+echo "私の名前は「 $name 」です";
 
 // Q2 四則演算
+$x = 5;
+$y = 4;
 
+$num = ($x*$y);
+
+var_dump($num);
+var_dump($num/2);
 
 // Q3 日付操作
 
+date_default_timezone_set('Asia/Tokyo');
+
+echo date("現在の時刻は、Y年m月d日 H時i分s秒です。");
 
 // Q4 条件分岐-1 if文
 
+$device = 'mac';
+
+if ($device === 'windows' ||  $device === 'mac') {
+    echo "使用OSは、 $device です。";
+} else {
+    echo "どちらでもありません。";
+}
+
 
 // Q5 条件分岐-2 三項演算子
 
+$age = 17;
+
+$message = ($age > 17) ? '成人です。' : '未成年です。';
+
+echo $message;
 
 // Q6 配列
 
+$prefecture = ['東京都', '神奈川県', '埼玉県','栃木県','千葉県','茨城県','群馬県'];
 
+echo "$prefecture[3]と$prefecture[4] は関東地方の都道府県です。";
 // Q7 連想配列-1
 
+$prefecture = ['東京都' => '新宿区','神奈川県' => '横浜市','千葉県' => '千葉市','埼玉県' => 'さいたま市','栃木県' => '宇都宮市','群馬県' => '前橋市','茨城県' => '水戸市'];
```

Comment:
```suggestion

$prefecture = [
　'東京都' => '新宿区',
　'神奈川県' => '横浜市',
　'千葉県' => '千葉市',
　'埼玉県' => 'さいたま市',
　'栃木県' => '宇都宮市',
　'群馬県' => '前橋市',
　'茨城県' => '水戸市'
];
```

Gitは行で変更を感知するため、1行が長くなる場合は改行しましょう！
（変更を加えた場合その行のどの部分が変わったのかが分かりづらいため）

Reply:
修正確認済み